app-frontend/src/app/pages/projects/edit/browse/browse.controller.js

```diff
@@ -50,7 +51,10 @@ export default class ProjectAddScenesBrowseController {
             this.getProjectSceneIds();
             this.initWatchers();
             this.initMap();
-            this.requestNewSceneList();
+            if (this.queryParams && this.queryParams.dataRepo
```

Comment:
you don't need the second check - if `dataRepo` doesn't exist, it won't = 'Raster Foundry'